molgenis/molgenis-emx2
# feat: aggregation dashboard

Due to merge conflicts and rebase issues with #3376, a new branch was created and this PR was opened. 

What are the main changes you did:
- Addition of aggregation dashboard that works with the table Catalogue/Aggregation.
- Added table for metastasis
- Added filter for resources
- Added page header

how to test:
- this UI is accessible in any database that uses the catalogue schema: url = `catalogue-demo/aggregation-dashboard`


Input needed / notes:
- Is it better to create a generic page or would you like it to be branded for a project? Is there a better name for the page?
- Please review chart titles. Do they make sense? Is there a better way to phrase the charts?
- Sample type is no longer part of the model, right? (I've removed this since it wasn't in the new datamodel)
- Should I add a section on "how to use the dashboard"?
- I'm not using the MolgenisVizEmx components. I still need to expand the library and enable filters. For now, I will continue with the current app structure.

Todo:
- [x] adjusted left margin on research center chart
- [x] when a row in primary tumor site table is clicked, the filters aren't generated
- [x] Update chart titles
- [x] fix bug where filters generated by cases by primary tumor chart disappear
- [x] remove on click event that updates charts when filter are applied
- [x] fix sorting of pie chart data
- ~~fix gap around app~~: needs a new PR
